library/src/main/java/com/google/android/exoplayer/text/eia608/ClosedCaptionCtrl.java

```diff
@@ -15,6 +15,9 @@
  */
 package com.google.android.exoplayer.text.eia608;
 
+import android.graphics.Color;
+import android.util.Log;
```

Comment:
Unused, as is TAG below. There's normally a setting you can enable in your IDE to highlight unused stuff, so that you can get rid of it.